profiles/__init__.py

```diff
@@ -68,47 +77,48 @@ def results(uid=None, info=None):
 def search(searched=None, info=None):
     # return jsonify(ldap_search_members(searched))
     searched = request.args.get("q")
-    return render_template("listing.html", 
-    						  info=info, 
-    						  title = "Search Results: "+searched,
+    return render_template("listing.html",
+    						  info=info,
+    						  title="Search Results: "+searched,
     						  members=ldap_search_members(searched))
 
 
-@app.route("/group/<group>", methods=["GET"])
+@app.route("/group/<_group>", methods=["GET"])
 @auth.oidc_auth
 @before_request
-def group(group=None, info=None):
-    group_desc = ldap_get_group_desc(group)
+def group(_group=None, info=None):
+    group_desc = ldap_get_group_desc(_group)
 
-    if "eboard" == group:
-    	return render_template("listing.html", 
+    if _group == "eboard":
+        return render_template("listing.html",
     						    info=info,
-    						    title = group_desc,
+    						    title=group_desc,
     						    members=ldap_get_eboard())
-    else:
-    	return render_template("listing.html", 
-    						    info=info, 
-    						    title = group_desc,
-    						    members=_ldap_get_group_members(group))
+
+    return render_template("listing.html",
+    						    info=info,
+    						    title=group_desc,
+    						    members=_ldap_get_group_members(_group))
 
 
 @app.route("/update", methods=["POST"])
 @auth.oidc_auth
 @before_request
 def update(uid=None, info=None):
     if request.method == "POST"  and 'photo' in request.files:
```

Comment:
There's no need to check if "POST" if you've only allowed "POST" in your methods list

Reply:
fixed them